modules/runner/src/integrationTest/java/org/apache/ignite/internal/distribution/zones/ItDistributionZonesFilterTest.java

```diff
@@ -95,6 +96,7 @@ protected int initialNodes() {
      *
      * @throws Exception If failed.
      */
+    @Disabled("https://issues.apache.org/jira/browse/IGNITE-19506")
```

Comment:
I would be grateful if you explain how your changes caused this test to be muted, and why they was muted with https://issues.apache.org/jira/browse/IGNITE-19506?

Reply:
My changes altered load profile (we don't write to vault anymore, so most metastorage operations should be faster) and it seems like it caused race condition with BaselineManager#nodes. We can observe incorrect topology and cause incorrect distribution, which in turn may cause a start of the duplicate raft group node

Reply:
I see that test fails when we check data nodes in metastorage after we start node with index 2. How is this connected with `BaselineManager#nodes`?